pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -580,6 +580,7 @@ func (rc *reconciler) reconstructVolume(volume podVolume) (*reconstructedVolume,
 		// volume.volumeSpecName is actually InnerVolumeSpecName. It will not be used
 		// for volume cleanup.
 		// TODO: in case pod is added back before reconciler starts to unmount, we can update this field from desired state information
+		// Fixed TODO with Issue#103143
```

Comment:
Maybe remove TODO, instead comment
```
in case pod is added back to desired state, outerVolumeSpecName will be updated from dsw information. See issue #103143 and its fix for details